frontend/src/components/worksheets/items/ImageItem.js

```diff
@@ -22,12 +22,12 @@ class ImageItem extends React.Component {
         var src = 'data:image/png;base64,' + this.props.item.image_data;
         var styles = {};
         if (this.props.item.hasOwnProperty('height')) {
-            styles['height'] = this.props.item.height + 'px;';
+            styles['height'] = this.props.item.height + 'px';
         }
         if (this.props.item.hasOwnProperty('width')) {
-            styles['width'] = this.props.item.width + 'px;';
+            styles['width'] = this.props.item.width + 'px';
         }
-
+        
```

Comment:
Can you remove this space?
```suggestion

```

Reply:
The space was originally here actually. Accidentally deleted it so added it back, it separates the code block & return block so I think we should keep it

Reply:
It wasn't originally there. Only the newline was there, but the tab wasn't. I meant to remove the tab, but keep the newline.

Reply:
Oh I see, I will fix this in a new pull request, sorry about that

Reply:
Removing it #1448 , will be more careful next time